tests/local-testnet/deploy.sh

```diff
@@ -49,6 +54,7 @@ do
     --unsafe-ws-external \
     --rpc-methods unsafe \
     --rpc-cors all \
+    --execution $EXECUTION \
```

Comment:
What if `$EXECUTION` is empty

Reply:
`$EXECUTION` comes from command, If empty, line 16~18, `$EXECUTION=native`

Reply:
Empty should be default behavior.

Reply:
@AurevoirXavier Please review again